notebooks/gauss_plot.ipynb

```diff
@@ -0,0 +1,150 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Plot pdf and cdf of standard normal
</pre><p>Reuse some code by creating a function. There is a repeatation of code here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/patel-zeel/pyprobml-fork/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>